include/EffectInfo.h

```diff
@@ -51,10 +59,19 @@ namespace openshot
 		// Create an instance of an effect (factory style)
 		EffectBase* CreateEffect(string effect_type);
 
+		// Load effect built in shared library (factory style)
+		EffectBase* LoadEffect(string location);
```

Comment:
Since the `using namespace std;` will be going away in #296, this should also be `std::string` (and throughout).

Reply:
Done